src/checks/noDependencyProblems/noUnusedDependencies.ts

```diff
@@ -6,6 +6,7 @@ import preflightBinPath from '../../util/preflightBinPath';
 export const title = 'No unused dependencies';
 
 export default async function noUnusedAndMissingDependencies() {
+  const ignoredFilePatterns = ['*.module.scss'].join(',');
```

Comment:
- maybe this should come after the `ignoredPackagePatterns`?
- `File` in the name doesn't seem to make sense, I guess this would not only be about files in the future?
- is `ignoredPackagePatterns` incorrectly named as well? (eg. if the `--ignores` flag doesn't actually take patterns)

Reply:
hm, maybe `ignoredFilePatterns` should be `ignoredPathPatterns`, seems closer to [the wording in the docs](https://github.com/upleveled/preflight/pull/448#discussion_r1342921348)

Reply:
Changed the order and moved the `ignoredPathPatterns` after the `ignoredPackagePatterns` and changed the variable name
[a11846f](https://github.com/upleveled/preflight/pull/448/commits/a11846fdbd17b60f3dcffeba732b922864f109d2)